pipeline/outputs/s3.md

```diff
@@ -40,7 +40,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
-| compression                      | Compression type for S3 objects. 'gzip' is currently the only supported value by default. If Apache Arrow support was enabled at compile time, you can also use 'arrow'. For gzip compression, the Content-Encoding HTTP Header will be set to 'gzip'. Gzip compression can be enabled when `use_put_object` is 'on' or 'off' (PutObject and Multipart). Arrow compression can only be enabled with `use_put_object On`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | None                                    |
+| compression                      | Compression type for S3 objects. 'gzip' and 'parquet' are currently the only supported value by default. If Apache Arrow support was enabled at compile time, you can also use 'arrow'. If columnify command is installed, you can also compress as parquet format. For gzip compression, the Content-Encoding HTTP Header will be set to 'gzip'. Gzip and parquet compression can be enabled when `use_put_object` is 'on' or 'off' (PutObject and Multipart). Arrow compression can only be enabled with `use_put_object On`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | None                                    |
```

Comment:
Is `none` a supported type or is it empty?

What happens if you do not set the `use_put_object` option appropriately - does it report a config failure (good) or just fail silently to do what you requested (bad)?

Reply:
I supposed that it means empty. But all of the documentation of out_s3 uses this habit of usage for None.

Reply:
> What happens if you do not set the use_put_object option appropriately - does it report a config failure (good) or just fail silently to do what you requested (bad)?

This cause just configuration error:
https://github.com/fluent/fluent-bit/pull/8837/files#diff-33a08a0c450f5b99e5c77cd5a5178e09b5482f262721994256817c03ebd0cdfbR687